app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -7,4 +7,23 @@ def show
     session[:check_answers_seen] = true
     render "coronavirus_form/check_answers"
   end
+
+  def submit
+    submission_reference = reference_number
+
+    session[:reference_number] = submission_reference
+    FormResponse.create(form_response: session)
```

Comment:
I guess this is still a work in progress, but it would be cool to get some validations on the data we're accepting ( this is probably more fiddly with a jsonb format ), like text length etc.

Reply:
Yeah was thinking about this, do we do validation in the individual questions and here? I guess we should also revalidate as someone could modify the cookie or hit this endpoint directly

Reply:
Could do in a separate PR?

Reply:
OK sounds good. I'd like validation here if possible since we might mess up/forget validation elsewhere.